spec/requests/users_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /users' do
+    before(:example) { get users_path }
+
+    it 'page load successful' do
+      expect(response).to have_http_status(200)
+    end
+
+    it 'renders the index template' do
+      expect(response).to render_template('index')
+    end
+
+    it 'contains the correct text' do
+      expect(response.body).to include('List all users')
+    end
+  end
+
```

Comment:
* According to the project requirement 

> Make sure that for each action you check:

> If the response status was correct.
> If a correct template was rendered.
> If the response body includes correct placeholder text.
* Kindly ensure you write tests to check for all the above in all your actions as some are missing